domain/src/main/java/org/oppia/android/domain/platformparameter/PlatformParameterAlphaModule.kt

```diff
@@ -263,4 +269,28 @@ class PlatformParameterAlphaModule {
       LOWEST_SUPPORTED_API_LEVEL_DEFAULT_VALUE
     )
   }
+
+  @Provides
+  @NpsSurveyGracePeriodInDays
+  fun provideNpsSurveyGracePeriodInDays(
+    platformParameterSingleton: PlatformParameterSingleton
+  ): PlatformParameterValue<Int> {
+    return platformParameterSingleton.getIntegerPlatformParameter(
+      NPS_SURVEY_GRACE_PERIOD_IN_DAYS
+    ) ?: PlatformParameterValue.createDefaultParameter(
+      NPS_SURVEY_GRACE_PERIOD_IN_DAYS_DEFAULT_VALUE
+    )
+  }
+
+  @Provides
+  @NpsSurveyMinimumAggregateLearningTimeInATopicInMinutes
+  fun provideNpsSurveyMinimumAggregateLearningTimeInATopicInMinutes(
+    platformParameterSingleton: PlatformParameterSingleton
+  ): PlatformParameterValue<Int> {
+    return platformParameterSingleton.getIntegerPlatformParameter(
+      NPS_SURVEY_MINIMUM_AGGREGATE_LEARNING_TIME_IN_A_TOPIC_IN_MINUTES
+    ) ?: PlatformParameterValue.createDefaultParameter(
+      NPS_SURVEY_MINIMUM_AGGREGATE_LEARNING_TIME_IN_A_TOPIC_IN_MINUTES_DEFAULT_VAL
```

Comment:
(Optional, feel free to disregard) If we are naming other things in a way that ends in DEFAULT_VALUE, we might want to do the same thing here. Perhaps "aggregate" can be replaced with "total", or "minutes" with "mins", if line length is an issue?

Reply:
I updated this constant.